sound/soc/sof/intel/intel-pci.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ */
+
+#ifndef __SOF_INTEL_PCI_H
+#define __SOF_INTEL_PCI_H
+
+static inline int sof_pci_intel_probe(struct pci_dev *pci, const struct pci_device_id *pci_id)
```

Comment:
this function is called by reference, so it's never inlined. Let's move it into .c

Reply:
that's right, but I didn't know in which module this should belong, so I opted for duplication with an inline mechanism. I'll re-think about it.

Reply:
fixed now